### DISCUSSION TITLE
Resizing image file of chrome OS?

### DISCUSSION THREAD

**Discussion by Gajalshankar** [sebanc/brunch#1138] [Q&A]
Recently I installed Chrome OS using following method from the guide - "**Dual Boot ChromeOS with Windows on your HDD**".
After using Linux,VS Code & Android Emulator for development,no more storage space is left. 
I would like to know the possibility of resizing the image partition without losing any data.

**Comment by darethehair** [comment]
Does this link help?

https://github.com/sebanc/brunch/issues/237

Is it the "stateful_partition" that you are running out of room on?

**Reply by Gajalshankar** [reply]
yes, it's a stateful_partition
